db/migrations/20220512132042_create_repository_configurations_table.up.sql

```diff
@@ -9,8 +9,7 @@ CREATE TABLE IF NOT EXISTS repository_configurations(
     account_id VARCHAR (255) NOT NULL,
     org_id VARCHAR (255) NOT NULL,
     created_at timestamp NOT NULL,
-    updated_at timestamp NOT NULL,
-    deleted_at timestamp
```

Comment:
i removed deleted_at as we discussed doing real deletes, and this was getting in the way of my testing